src/v2/view-builder/views/TerminalView.js

```diff
@@ -105,7 +105,10 @@ const Body = BaseForm.extend({
 
       messagesObjs.value
         .forEach(messagesObj => {
-          const msg = messagesObj.message;
+          let msg = messagesObj.message;
+          if (messagesObj.i18n?.key === OPERATION_CANCELED_BY_USER_KEY) {
+            msg = loc(messagesObj.i18n.key, 'login');
+          }
```

Comment:
I see this growing for similar occurrences. Here's another idea I have that involves a courage change but cleaner imho.

* Currently `loc` function `return`s an error string like `L10N[blah.blah]` not found. Ref - https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/StringUtil.js#L158
* Instead of that, we could change the `loc` function to accept another argument (say `throwErrorOnFailure`) and change it to -
```
localize: function (key, bundleName, params, throwErrorOnFailure) {
  .
  .
  .
  try {
    params = params && params.slice ? params.slice(0) : [];
    params.unshift(value);
    value = StringUtil.sprintf.apply(null, params);
  } catch (e) {
    value = null;
  }
  if (throwErrorOnFailure) {
    throw new Error( 'L10N_ERROR[' + key + ']');
    return;
  }
  return value || 'L10N_ERROR[' + key + ']';
```

* This enables me to do something like this in my code instead -
```
if (messagesObj.i18n?.key) {
  try {
    msg = loc(messagesObj.i18n.key, 'login', [], true) || messagesObj.message;
  } catch(e) {
    // do nothing.
  }
}
```